### ISSUE TITLE
New page: pick_an_image.md

### ISSUE THREAD

**Issue by why-not-try-calmer** [openSUSE/openSUSE-docs-revamped#37]
Opening discussion and reviewing after #36 .

**Comment by apinter** [comment]
IMO this is very well put together, other than the little typo under `Sway` installation steps. It says "install LXQT". 
Do we want to include a picture for each variant tho?

**Comment by workingwriter** [comment]
Comments on Choosing a Distribution

Tumbleweed section
    "you will be frequently offered to upgrade ..." - Add "sometimes several times a week" Make it clear why you need to have that "reliable internet connection."
    
    I'm guessing that we want a graphic/icon for the "???Info" paragraphs? That would be good.
    
Style: Avoid using "i.e." and "e.g" in documentation. People often confuse the two, and few people know what Latin words these abbreviations take the place of.
    i.e. = "That is"
    e.g. = "for example" or "such as"

Also fix typo: "(<del>i.e.</del>That is, those that constitute **your** Linux distribution).

TIP: Wherever possible, use ACTIVE VOICE -- avoid sentences with all the various forms of "to be."
    For example: 
    "Even though Tumbleweed uses a rolling-release model, openSUSE thoroughly tests every package before going into an update, ensuring the stability of your system. You can always revert back to the previous state of Tumbleweed if something goes wrong."

"Snapshots are explained here" (needs real link) 

Leap section
    Include the time interval between point releases? 
    TYPO: SUSE Linux **Enterprise**
    
    Suggestion:
    "The kernel is the heart of any Linux operating system. It translates your keystrokes and mouse clicks into the ones and zeros the computer understands, and is updated regularly so that the computer can respond to new hardware added to the system." 
    
Recommendation section
    Blockers
    - You <del>don't want or you cannot update frequently over the network, because you</del> have a very limited bandwidth or unreliable internet access.
    - You cannot afford to reboot your computer frequently to install updates.
    - You don't want to use, or <del>for some reason cannot afford to</del> switch between<del>,</del> different snapshots.
    - You<r> <del>have an old</del> computer **is** more than 7 years old **and** <del>which</del> cannot take advantage of recent kernels.
    - You **know exactly what** <del>have a fixed set of</del> packages <del>you know</del> you are going to install once and forget about, and **you don't use** <del>usually you never want to have anything to do with</del> command line utilities.
    
Info about microOS: Consider this rewrite:
Advanced users might consider installing microOS on friends and relatives computers as an introduction to Linux. This OS requires no maintenance from them, and very little maintenance for you. See the <link> microOS page</link> for more about this use case.

Desktop Environments

New intro paragraph:
"Unlike Windows or Macs, Linux offers a variety of desktop choices, and each offers a different look-and-feel, and built-in functionality. In this section, you'll learn about the different environments to help you choose a default when you install openSUSE. You'll be asked to select a desktop in the **System Role** section of the installation. You can always install another desktop environment using YaST if you don't like your first choice."

/* Might be nice to have screenshots of each DE */

Why is Cinnamon by itself under Advanced Options?

**Comment by why-not-try-calmer** [comment]
Thanks a lot Mike, I have tried to accomodate most of your comments on #84
